gtm.go

```diff
@@ -276,6 +276,124 @@ func (b *BigIP) DeleteGTMAPoolMember(fullPathToAPool, serverFullPath, poolMember
 	return b.delete(uriGtm, uriPool, string(ARecord), fullPathToAPool, uriPoolMember, fullPathToPoolMember)
 }
 
+// ********************************************************************************************************************
+// *******************************************                    *****************************************************
+// *******************************************   GTM Pool CNAME   *****************************************************
+// *******************************************                    *****************************************************
+// ********************************************************************************************************************
```

Comment:
Thank you for the commit! Would you mind removing the banner comments thought, as it will just keep the normal documentation consistent. Thank you!

Reply:
Done, thank you